src/lib/repositories_v2/generic_repository.py

```diff
@@ -0,0 +1,25 @@
+# Interface for the repositories through Abstract method
+
+from abc import ABC, abstractmethod
+
+
+class GenericRepository(ABC):
```

Comment:
Why don't you use the existing generic repository?

Reply:
done.